src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -355,6 +382,60 @@
         }
       }
     },
+    "notification": {
+      "type": "object",
+      "description": "Describes a condition relevant to the tool itself, as opposed to being relevant to a file being analyzed by the tool.",
+      "properties": {
+        "id": {
+          "description": "A stable, unique identifier for the condition that was encountered.",
```

Comment:
Fixed.

---
In reply to: [61636572](https://github.com/Microsoft/sarif-sdk/pull/229#discussion_r61636572) [](ancestors = 61636572,61636508)